gatsby-site/src/components/blog-article/BlogArticle.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { articles } from '../blog-list/BlogList.js';
+import styles from './BlogArticle.css'
+
+export default function BlogArticle(props) {
+  const path = window.location.pathname;
```

Comment:
This isn't going work when we try to server side render things. Could we use a prop to get this info?

Reply:
Sure thing! I’ll also take out the routing until I can route it in unstack.